go-controller/pkg/ovn/egressip.go

```diff
@@ -416,55 +416,94 @@ func (e *egressIPMode) needsRetry(pod *kapi.Pod) bool {
 // to retrive the internal gateway router IP attached to the egress node. This method handles both the shared and
 // local gateway mode case, and in the local case (i.e: packetMark != 0) uses the optional packetMark field
 // on the logical router policy to properly L3 SNAT on the node side.
-func (e *egressIPMode) createEgressPolicy(podIps []net.IP, status egressipv1.EgressIPStatusItem, packetMark uint32) error {
+func (e *egressIPMode) createEgressPolicy(podIps []net.IP, status egressipv1.EgressIPStatusItem, packetMark uint32, egressIPName string) error {
 	isEgressIPv6 := utilnet.IsIPv6String(status.EgressIP)
 	gatewayRouterIP, err := e.getGatewayRouterJoinIP(status.Node, isEgressIPv6)
 	if err != nil {
 		return fmt.Errorf("unable to retrieve node's: %s gateway IP, err: %v", status.Node, err)
 	}
 	for _, podIP := range podIps {
+		var err error
+		var stderr, filterOption string
 		if isEgressIPv6 && utilnet.IsIPv6(podIP) {
-			var stderr string
-			var err error
-			if packetMark == 0 {
-				_, stderr, err = util.RunOVNNbctl("lr-policy-add", ovnClusterRouter, egressIPRereoutePriority,
-					fmt.Sprintf("ip6.src == %s && ip6.dst == ::/0", podIP.String()), "reroute", gatewayRouterIP.String())
-			} else {
-				_, stderr, err = util.RunOVNNbctl("lr-policy-add", ovnClusterRouter, egressIPRereoutePriority,
-					fmt.Sprintf("ip6.src == %s && ip6.dst == ::/0", podIP.String()), "reroute", gatewayRouterIP.String(), fmt.Sprintf("pkt_mark=%v", packetMark))
-			}
-			if err != nil && !strings.Contains(stderr, policyAlreadyExistsMsg) {
-				return fmt.Errorf("unable to create logical router policy for egress IP: %s, stderr: %s, err: %v", status.EgressIP, stderr, err)
-			}
+			filterOption = fmt.Sprintf("ip6.src == %s && ip6.dst == ::/0", podIP.String())
```

Comment:
Hmm, in the beginning I added this as to avoid rerouting east<->west traffic, but at one moment I think I noticed that didn't work and thus added the no-reroute policies created by `createDefaultNoReroutePodPolicies` / `createDefaultNoRerouteNodePolicies`. So I think it's safe to remove this.